proto/viam/app/v1/app.proto

```diff
@@ -708,21 +712,30 @@ message RemoveRoleRequest {
 
 message RemoveRoleResponse {}
 
+message Authorizations {
+  string identity_id = 1;
+  repeated Authorization authorizations = 2;
+}
+
 message ListAuthorizationsRequest {
   string resource_type = 1;
   string resource_id = 2;
   string organization_id = 3;
 }
 
-message Authorizations {
-  string identity_id = 1;
-  repeated Authorization authorizations = 2;
-}
-
 message ListAuthorizationsResponse {
   repeated Authorizations authorizations = 1;
 }
 
+message ListAuthorizationsForUserRequest {
+  string user_id = 1;
+  string organization_id = 2;
+}
+
+message ListAuthorizationsForUserResponse {
+  repeated Authorization authorizations = 1;
```

Comment:
note that the return of this is a `repeated Authorization` instead of a `repeated Authorizations`. if i'm understanding correctly i think it's okay to omit the `identity_id` part of the `Authorizations` because these should all be for the same identity (the given user)